components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -831,62 +597,134 @@ subroutine micro_p3_init(pbuf2d)
-      call add_default('P3_mtend_CLDICE',  1, ' ')
-      call add_default('P3_mtend_NUMICE',  1, ' ')
-      call add_default('P3_mtend_Q',       1, ' ')
-      call add_default('P3_mtend_TH',      1, ' ')
+      if (micro_tend_output) then
+         call add_default('P3_qrcon',  1, ' ')
+         call add_default('P3_qcacc',  1, ' ')
+         call add_default('P3_qcaut',  1, ' ')
+         call add_default('P3_ncacc',  1, ' ')
+         call add_default('P3_ncautc', 1, ' ')
+         call add_default('P3_ncslf',  1, ' ')
+         call add_default('P3_nrslf',  1, ' ')
+         call add_default('P3_ncnuc',  1, ' ')
+         call add_default('P3_qccon',  1, ' ')
+         call add_default('P3_qcnuc',  1, ' ')
+         call add_default('P3_qrevp',  1, ' ')
+         call add_default('P3_qcevp',  1, ' ')
+         call add_default('P3_nrevp',  1, ' ')
+         call add_default('P3_ncautr', 1, ' ')
+         ! ice-phase process rates
+         call add_default('P3_qccol',  1, ' ')
+         call add_default('P3_qwgrth', 1, ' ')
+         call add_default('P3_qidep',  1, ' ')
+         call add_default('P3_qrcol',  1, ' ')
+         call add_default('P3_qinuc',  1, ' ')
+         call add_default('P3_nccol',  1, ' ')
+         call add_default('P3_nrcol',  1, ' ')
+         call add_default('P3_ninuc',  1, ' ')
+         call add_default('P3_qisub',  1, ' ')
+         call add_default('P3_qimlt',  1, ' ')
+         call add_default('P3_nimlt',  1, ' ')
+         call add_default('P3_nisub',  1, ' ')
+         call add_default('P3_nislf',  1, ' ')
+         call add_default('P3_qcheti', 1, ' ')
+         call add_default('P3_qrheti', 1, ' ')
+         call add_default('P3_ncheti', 1, ' ')
+         call add_default('P3_nrheti', 1, ' ')
+         call add_default('P3_nrshdr', 1, ' ')
+         call add_default('P3_qcshd',  1, ' ')
+         call add_default('P3_ncshdc', 1, ' ')
+         ! Sedimentation
+         call add_default('P3_sed_CLDLIQ',  1, ' ')
+         call add_default('P3_sed_NUMLIQ',  1, ' ')
+         call add_default('P3_sed_CLDRAIN', 1, ' ')
+         call add_default('P3_sed_NUMRAIN', 1, ' ')
+         call add_default('P3_sed_CLDICE',  1, ' ')
+         call add_default('P3_sed_NUMICE',  1, ' ')
+         ! Microphysics Processes
+         call add_default('P3_mtend_CLDLIQ',  1, ' ')
+         call add_default('P3_mtend_NUMLIQ',  1, ' ')
+         call add_default('P3_mtend_CLDRAIN', 1, ' ')
+         call add_default('P3_mtend_NUMRAIN', 1, ' ')
+         call add_default('P3_mtend_CLDICE',  1, ' ')
+         call add_default('P3_mtend_NUMICE',  1, ' ')
+         call add_default('P3_mtend_Q',       1, ' ')
+         call add_default('P3_mtend_TH',      1, ' ')
+      end if
    end if
 
+!    call addfld ('PRODPREC', (/ 'lev' /), 'A', 'kg/kg/s', 'Rate of conversion of condensate to precip'              )
+!    call addfld ('EVAPPREC', (/ 'lev' /), 'A', 'kg/kg/s', 'Rate of evaporation of falling precip'                   )
```

Comment:
Remove?

Reply:
Yeah, we could probably remove this.  We aren't fully done with the code cleanup but we wanted to get some recent changes in so that Peter B could use them for his SHOC+P3+RRTGMP runs.